modules/javafx.graphics/src/main/java/com/sun/glass/ui/gtk/GtkApplication.java

```diff
@@ -467,4 +469,40 @@ protected boolean _supportsInputMethods() {
     @Override
     protected native int _isKeyLocked(int keyCode);
 
+    @Override
+    public native Map<String, Object> getPlatformPreferences();
+
+    @Override
+    public Map<String, String> getPlatformKeyMappings() {
+        return Map.of(
+            "GTK.theme_fg_color", "foregroundColor",
+            "GTK.theme_bg_color", "backgroundColor"
+        );
+    }
+
+    // This list needs to be kept in sync with PlatformSupport.cpp in the Glass toolkit for GTK.
+    @Override
+    public Map<String, Class<?>> getPlatformKeys() {
+        return new HashMap<>() {{
```

Comment:
Any reason this map and the like are not `Map.ofEntries`? The double braces syntax is not a good idea in general as it creates a subclass of `HashMap` for just one initialization.

Reply:
I've replaced `HashMap` with `Map.ofEntries` as you've suggested.